docs/source/algorithms/Stitch1DMany-v1.rst

```diff
@@ -13,22 +13,72 @@ Stitches single histogram :ref:`Matrix Workspaces <MatrixWorkspace>`
 together outputting a stitched Matrix Workspace. This algorithm is a
 wrapper over :ref:`algm-Stitch1D`.
 
-The algorithm expects pairs of StartOverlaps and EndOverlaps values. The
-order in which these are provided determines the pairing. There should
-be N entries in each of these StartOverlaps and EndOverlaps lists, where
-N = 1 -(No of workspaces to stitch). StartOverlaps and EndOverlaps are
-in the same units as the X-axis for the workspace and are optional.
-
-The workspaces must be histogrammed. Use
-:ref:`algm-ConvertToHistogram` on workspaces prior to
-passing them to this algorithm.
+The workspaces must be histogrammed. Use :ref:`algm-ConvertToHistogram` on
+workspaces prior to passing them to this algorithm.
+
+The algorithm expects pairs of :literal:`StartOverlaps` and
+:literal:`EndOverlaps` values. The order in which these are provided determines
+the pairing. There should be N entries in each of these lists, where N = 1 - (No
+of workspaces to stitch). StartOverlaps and EndOverlaps are in the same units as
+the X-axis for the workspace and are optional. For each pair of these values, the
+:literal:`StartOverlaps` value cannot exceed its corresponding
+:literal:`EndOverlaps` value. Furthermore, if either the start or end value is
+outside the range of X-axis intersection, they will be forcibly changed to the
+intersection min and max respectively.
 
 This algorithm is also capable of stitching together matrix workspaces
 from multiple workspace groups. In this case, each group must contain the
 same number of workspaces. The algorithm will stitch together the workspaces
 in the first group before stitching workspaces from the next group on top
 of the previous ones.
 
+When stitching the workspaces, either the RHS or LHS workspaces can be scaled.
+We can specify a manual scale factor to use by setting
+:literal:`UseManualScaleFactor` true and passing a value to
+:literal:`ManualScaleFactor`. For group workspaces, we can also use
+:literal:`ScaleFactorFromPeriod` to select a period which will obtain a vector
+of scale factors from the selected period. These scale factors are then applied
+to all other periods when stitching.
+
+Workflow
+--------
+
+The algorithm workflow is as follows:
+
+#. A check is performed to find out whether the input workspaces are group
+   workspaces or not. The algorithm handles matrix workspaces differently from
+   group workspaces.
+#. If matrix workspaces are supplied, the algorithm simply iterates over each
+   workspace and calls the :literal:`Stitch1D` algorithm, each RHS workspace to
+   the LHS workspace to form a single stitched workspace. The resultant
+   workspace and its scale factor are outputted.
+#. If group workspaces are supplied, the algorithm checks whether a global scale
```

Comment:
I should probably rewrite this to make it more clear (the comments in the code were in fact wrong). There's no global scale factor, rather that if we are not using `ScaleFactorFromPeriod`, the workspaces for each period across all groups will be stitched together using Stitch1D. `UseManualScaleFactor` and `ManualScaleFactor` will be passed to them, but `UseManualScaleFactor` may not necessarily be true.